lib/chef/provider/mount/mount.rb

```diff
@@ -203,7 +203,8 @@ def device_real
             end
           end
           # Removed "/" from the end of str, because it was causing idempotency issue.
-          @real_device.chomp("/")
+          # On AIX, device property value can be "/"
+          !RUBY_PLATFORM.match?(/aix/i) ? @real_device.chomp("/") : @real_device
```

Comment:
```suggestion
          # On AIX, device property value can be "/"
          @real_device == "/" ? @real_device : @real_device.chomp("/")
```

I know this has only come up on AIX, but you could make this more general purpose in case it ever happens in another scenario.

Also, adding a test for this scenario would probably be a good idea.

Reply:
Agreed, I have fixed it.